src/memory/client/MemoryBrowser.tsx

```diff
@@ -77,21 +77,50 @@ interface State {
   bytesPerRow: number;
   bytesPerGroup: number;
   endianness: 'le' | 'be';
+  childrenNames: { id: number; name: string }[];
 }
 
 export class MemoryBrowser extends React.Component<Props, State> {
   private addressReq = '';
   private lengthReq = '512';
+  private childReq = 0;
+  public static firstTime: boolean = true;
 
   constructor(props: Props) {
     super(props);
     this.state = {
       bytesPerRow: 32,
       bytesPerGroup: 8,
       endianness: 'le',
+      childrenNames: [],
     };
   }
 
+  async getAvailableChilds() {
+    try {
+      const result = await messageBroker.sendGetChildrenNames({
+        command: 'getChildDapNames',
+        args: {
+          child: this.childReq,
```

Comment:
This argument does not make sense to me. What does a childReq mean in the context of getting all the children?

Reply:
I deleted it. Because we just need to post a comment cdt-amalgamator/getChildDapNames and get a response. It is not necessary to enclose args.